packages/sdk/test/integration/history_tree_split_test.ts

```
@@ -28,21 +27,24 @@ function topRedoTreeEdit(
   }
   return undefined;
 }
+// Renders a reverse op's identity payload. Reverse ops address nodes by
+// identity rather than carrying a subtree copy, so this reads the span sets
+// rather than `contents`.
 function summarizeOp(op: Operation | undefined): string {
   if (!(op instanceof TreeEditOperation)) return '<not-tree-edit>';
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)